integrations/graal/mp-native-image-extension/src/main/java/io/helidon/integrations/graal/mp/nativeimage/extension/WeldSubstitutions.java

```diff
@@ -92,12 +94,12 @@ public static void set(Object object, Executor executor) {
 
         }
     }
-// TODO 3.0.0-JAKARTA
-//    @TargetClass(className = "org.jboss.weld.util.bytecode.ClassFileUtils")
-//    static final class ClassFileUtils {
-//        @Substitute
-//        public static Class<?> toClass(ClassFile ct, ClassLoader loader, ProtectionDomain domain) {
-//            throw new IllegalStateException("Cannot load " + ct.getName());
-//        }
-//    }
+
+    @TargetClass(className = "org.jboss.weld.util.bytecode.ClassFileUtils")
```

Comment:
The reason this was commented out is because the file no longer exists in Weld

Reply:
So this failed during native image build

Reply:
Remove internal class `ClassFileUtils` from `WeldSubstitutions`